client/src/utils/consts.js

```diff
@@ -17,4 +17,95 @@ export const requiredFields = [
   'phone',
 ];
 
-export default { levelEnum, requiredFields };
+export const chapterOptions = [
+  {
+    key: 'University of Pennsylvania',
+    value: 'University of Pennsylvania',
+    text: 'University of Pennsylvania',
+  },
+  {
+    key: 'University of Illinois at Urbana-Champaign',
+    value: 'University of Illinois at Urbana-Champaign',
+    text: 'University of Illinois at Urbana-Champaign',
+  },
+  {
+    key: 'Bits of Good - Georgia Tech',
+    value: 'Bits of Good - Georgia Tech',
+    text: 'Bits of Good - Georgia Tech',
+  },
```

Comment:
this is fine as is! But just fyi as a nit if you wanted you could make shorter like:
`const chapters =[ 'University of Pen', University of Illinois', 'Bits of Good - Georgia Tech',  ...etc]`
and then 
`export const  chapterOptions = chapters.map((chapter) => ({ key: chapter, value: chapter, text: chapter })`